### PR TITLE
Reset style button with Dark style option!

### PR DESCRIPTION
**Pull request by ShellyHerself**
Also redoes some padding and font sizes and choices to be easier on people's eyes.

### CODE REVIEW

File: widgets/binilla_widget.py [hunk 1]
```diff
@@ -13,140 +13,177 @@
...
+        if dark:
+            cls.ttk_theme = "clam"
+        else:
+            cls.ttk_theme = "alt"
+
+        cls.read_traces = ()
+        cls.write_traces = ()
+        cls.undefine_traces = ()
+
+        cls._filedialog_style_fix = None
```
Comment (MosesofEgypt) [inline widgets/binilla_widget.py]: Do not set _filedialog_style_fix to None. Let it be defined as None in BinillaWidget, and let it be properly initialized in the \_\_init\_\_ method. The _filedialog_style_fix should only ever be instantiated ONCE in the lifetime of binilla. That should fix fix_filedialog_style breaking, as you commented it out below.

**Review by MosesofEgypt — CHANGES_REQUESTED** [review]
Check the comment

**Comment by ShellyHerself** [comment]
Updated, I was resetting too much, so I moved some stuff out of the class method.

**Review by MosesofEgypt — CHANGES_REQUESTED** [review]
I have thought about it, and set_style_defaults should not be a classmethod, it should be a staticmethod. The reason for this is that each class is not intended to contain its own defaults for the style values, but making it a classmethod implies otherwise. The BinillaWidget class itself is the single place these defaults should exist. I recommend making it a staticmethod, where "cls" is replaced with "BinillaWidget".

```python
    @staticmethod
    def set_style_defaults(dark=False):
        ...
        if dark:
            BinillaWidget.ttk_theme = "clam"
        else:
            BinillaWidget.ttk_theme = "alt"
```

**Pull request merged**